reference/pages.v3.yml

```diff
@@ -583,6 +581,15 @@ components:
           type: integer
       in: query
       required: true
+    deleteChildrenQuery:
+      name: 'delete_children'
+      example: true
+      description: |
+        When this query parameter is explicitly set to `true`, deleting a parent page will recursively delete all its immediate children and their descendants.
+        Otherwise, if this query parameter is set to `false` or not provided, deleting a parent page will update its immediate children by setting their `parent_id` to `0` and their `is_visible` status to `false`.

```

Comment:
```suggestion
        Otherwise, if you set this query parameter to `false` or not provided, deleting a parent page will update its immediate children by setting their `parent_id` to `0` and their `is_visible` status to `false`.

```

Reply:
Done